File: wrappers/java/src/main/java/com/evernym/vdrtools/wallet/WalletEncryptionException.java
package com.evernym.vdrtools.wallet;

import com.evernym.vdrtools.ErrorCode;
import com.evernym.vdrtools.IndyException;

/**
 * Exception thrown occurred during encryption-related operations.
 */
public class WalletEncryptionException extends IndyException
{
	private static final long serialVersionUID = 1829076830401150667L;
	private final static String message = "Error during encryption-related operations.";

	/**
	 * Initializes a new WalletEncryptionException.
	 */
	public WalletEncryptionException()
	{
		super(message, ErrorCode.WalletEncryptionError.value());
	}
}

File: wrappers/dotnet/indy-sdk-dotnet/WalletApi/WalletEncryptionException.cs
namespace Hyperledger.Indy.WalletApi
{
    /// <summary>
    /// Exception thrown when an error occurred during encryption-related operations.
    /// </summary>
    public class WalletEncryptionException : IndyException
    {
        /// <summary>
        /// Initializes a new WalletEncryptionException.
        /// </summary>
        internal WalletEncryptionException(string message) : base(message, (int)ErrorCode.WalletEncryptionError)
        {

        }
    }

}
